reproman/support/jobs/job_templates/cluster/condor_config.local.j2

```diff
@@ -0,0 +1,56 @@
+# THIS FILE IS CONTROLLED BY ELASTICLUSTER
```

Comment:
setup.py should be adjusted so that this file is included in `package_data`.

---

I see you've gone with ansible's convention of using the `.j2` extension for jinja templates.  Not something that should be done in this PR, but perhaps I should change all the other template files to do the same.

Reply:
@kyleam 
Actually, why don't I move this .j2 extension to .template to be consistent with the other templates. We are not actually using Ansible. I think I did that because, at the time I created the first version of the template, I was trying to integrate elasticluster which was using Ansible. Elasticluster uses the .j2 extension.

Reply:
@mjtravers Sounds good to me.